plugins/polio/js/src/domains/Chronogram/ChronogramDetails/Table/useChronogramDetailsTableColumn.tsx

```diff
@@ -63,8 +63,7 @@ export const useChronogramDetailsTableColumn = (
             {
                 Header: formatMessage(MESSAGES.labelUserInCharge),
                 id: 'user_in_charge',
-                accessor: row =>
-                    row.user_in_charge.full_name || row.user_in_charge.username,
+                accessor: 'user_in_charge',
```

Comment:
```suggestion
                accessor: 'user_in_charge',
                Cell: BreakWordCell
```

This would make sure long names are automatically wrapped. Don't forget to add the import if you use that suggestion

Reply:
Actually nevermind, it doesn't change anything